mitiq/lre/tests/test_lre.py

```diff
@@ -55,6 +71,29 @@ def test_lre_exp_value(degree, fold_multiplier):
     assert abs(lre_exp_val - ideal_val) <= abs(noisy_val - ideal_val)
 
 
+@pytest.mark.parametrize(
+    "degree, fold_multiplier, observable",
+    [(2, 2, obs_z), (2, 3, obs_z), (3, 4, obs_z)],
+)
```

Comment:
Can we make one of the tests on a setup (circuit and observable) that's on two qubits? There could be something that breaks when more qubits are introduced.

Reply:
I removed two of the single qubit tests cases and simplified the 2 qubit case in https://github.com/unitaryfund/mitiq/pull/2681/commits/e7986ab29dc5f6b65bb9ab8bf678cc2ff8c461fe just so it could run faster. Crazy that it takes 15 minutes! Once CI goes green on this I'll merge it!